### ISSUE TITLE
Installation Docs

### ISSUE THREAD

**Issue by DiegoGallegos4** [Jpadilla1/react-native-ios-charts#12]
Hi Jose,

Great work with this library. This is a request to improve the installation steps posted, most of them are very abstract and not easy to follow. I get to the end but there are two errors: 
1. CocoaPods
   ![image](https://cloud.githubusercontent.com/assets/5933437/14063587/8fae4a02-f395-11e5-996a-0f954887d528.png)
2. Xcode: 

![image](https://cloud.githubusercontent.com/assets/5933437/14063590/a5fe2f66-f395-11e5-80b5-5a0c0c97274f.png)

I already did the following:
1. All the files on the RNiOSCharts directory I copied them to the ios folder.
2. The bridging header file is created
3. When I run pod install i got the error on (1)
4. I already changed Embedded content contains swift to Yes. 

After this I still get on Chrome debugger: 

![image](https://cloud.githubusercontent.com/assets/5933437/14063593/0e9b4414-f396-11e5-9174-2efeb29ca391.png)

Cheers

**Comment by Jpadilla1** [comment]
Hi @DiegoGallegos4 I absolutely agree on this. I'll focus on getting everything setup as easy as possible as well as easy to follow along.

Regarding your issue, after you ran `pod install`, did you open the `.xcworkspace` file instead of the `.xcodeproject`?

Also, have you read issue #5 and see if it's the same issue happening to you?

**Comment by DiegoGallegos4** [comment]
Jose, thanks for the quick reply. If it is of any help I downloaded new version of Xcode(7.3) and Swift (2.2) today. Check [this](https://github.com/SwiftyJSON/SwiftyJSON/issues/482).

I saw that issue. Ill add something to the installation: 
1. Adding `${inherited}` to **Build Settings > Linking > Other Linker Flags** 

Now, I have this error: (Which I presume is due SwiftyJSON)

![image](https://cloud.githubusercontent.com/assets/5933437/14063875/0ad2ba56-f3a5-11e5-91b4-f35a42759625.png)

Chrome debugger error persists.

**Comment by L-Jovi** [comment]
maybe you could try this which solve your first issue.

Go to your target __Build Settings -> Other linker flags -> double click__ . Add `$(inherited)` to a new line.

**Comment by Jpadilla1** [comment]
Strange, I just downloaded and tried XCode 7.3 with swift 2.2 and didn't have any problems. I'll try to make a video or better docs this week and get back to you.

**Comment by DiegoGallegos4** [comment]
Hi Jose, I had to research how to remove CocoaPads and unlink the bridging header. After this, I tried again with the issue you linked for me to see and made it work. If I have time, I may take a couple of screenshots on the process to help out on the installation process for the library.

Cheers!

PD: There are some errors of syntax being deprecated on SwiftyJSON on Swfit 3. What other option do we have when that happens? Any help, i'd love to try.
